dist/s_string

```diff
@@ -45,9 +45,12 @@ check() {
 }
 
 # List of files to spellchk.
+# FIXME-WT-9433 Skip files in the test/cppsuite folder.
 l=`(cd .. &&
-    find bench examples ext src test -name '*.[chsy]' &&
-    find src -name '*.in' && find test -name '*.cxx' && find test -name '*.cpp')`
+    find bench examples ext src test -not -path "test/cppsuite/*" -name '*.[chsy]' &&
+    find src -name '*.in' &&
+    find test -name '*.cxx' &&
```

Comment:
I know this bit is just reformatted from the existing code, but I don't think we have any files matching this criteria any more after we moved everything to be `.cpp`, right? At least, `find` didn't get anything on a clean checkout.

Reply:
Agreed. Our only remaining .cxx files are 
```
./bench/workgen/workgen.cxx
./tools/xray_to_optrack/xray_to_optrack.cxx
```
neither of which are covered by the existing find statements.

Reply:
Fixed in [b4f8dfd](https://github.com/wiredtiger/wiredtiger/pull/8010/commits/b4f8dfd3b038235ada351b551823c66ca180a223)